.github/workflows/build_cmake.yml

```diff
@@ -60,7 +60,8 @@ jobs:
       # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
       run: |
         source ${{github.workspace}}/.github/CI/spack_setup.sh
-        cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $BUILD_CONFIG CC=gcc CXX=g++
+        echo CC=gcc CXX=g++ FC=gfortran cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $BUILD_CONFIG
+        CC=gcc CXX=g++ FC=gfortran cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $BUILD_CONFIG
```

Comment:
Maybe a nitpick, but isn't the canonical way of specifying the compiler in cmake to use `-DCMAKE_C_COMPILER=gcc` instead of `CC=gcc`?

Reply:
The environment variable takes precedence if the -DCMAKE_C_COMPILER is not specified. For all practical purposes, in our case the two are equivalent.